### PR TITLE
Check for local path if USB is not found

### LINKED ISSUE #76: encrypted saves from hdd
could you add a folder on the hdd where you can put ecnrpted usb saves so that you can copy saves from the internet directly to the console via ftp without having to plug in a usb

### PR DESCRIPTION
**Pull request by illusion0001**
Fixes #76

### CODE REVIEW

**Comment by illusion0001** [comment]
Fixes #76

**Comment by bucanero** [comment]
I'm thinking about a more simplistic approach.

Scan all USB ports, if nothing is found, then try the `/data/fakeusb/` folder. If that works, then use the fake usb folder as the folder for USB saves.

No need for new settings or anything.

File: source/main.c [hunk 1]
```diff
@@ -319,15 +320,22 @@ static int LoadSounds(void* data)
...
-	for (int i = 0; i <= MAX_USB_DEVICES; i++)
+	if(apollo_config.hddpath)
 	{
-		sprintf(path, USB_PATH "PS4/", i);
-
+		sprintf(path, APOLLO_PATH "PS4/");
 		if (dir_exists(path) == SUCCESS)
 			return;
 	}
-
```
Comment (bucanero) [inline source/main.c:329]: Just keep the original search here, then when out of the loop, the code can check
```c
if (dir_exists("/data/fakeusb/PS4/") == SUCCESS)
    sprintf(path, "/data/fakeusb/PS4/");
```

**Review by bucanero — CHANGES_REQUESTED** [review]

File: include/settings.h [hunk 2]
```diff
@@ -32,6 +32,7 @@ typedef struct
     uint8_t doSort;
     uint8_t doAni;
     uint8_t update;
+    uint8_t hddpath;
```
Comment (bucanero) [inline include/settings.h:35]: no need for new settings or callbacks

File: source/main.c (hunk 1 above)
Comment (illusion0001) [reply source/main.c]: Fixed

File: include/settings.h (hunk 2 above)
Comment (illusion0001) [reply include/settings.h]: Reverted

**Comment by illusion0001** [comment]
Tested, seems to work 👍

**Review by bucanero — APPROVED** [review]

**Pull request merged**